tests/Auth0.AuthenticationApi.IntegrationTests/PasswordlessTests.cs

```diff
@@ -106,5 +115,167 @@ public async Task Can_launch_sms_flow()
                 response.PhoneNumber.Should().Be(request.PhoneNumber);
             }
         }
+
+        [Fact(Skip = "Run manually")]
+        public async Task Can_exchange_sms_code_for_access_token()
+        {
+            var authenticationApiClient = new AuthenticationApiClient(GetVariable("AUTH0_AUTHENTICATION_API_URL"));
+
+            // Exchange the code
+            var token = await authenticationApiClient.GetTokenAsync(new PasswordlessSmsTokenRequest
+            {
+                ClientId = GetVariable("AUTH0_PASSWORDLESSDEMO_CLIENT_ID"),
+                ClientSecret = GetVariable("AUTH0_PASSWORDLESSDEMO_CLIENT_SECRET"),
+                Code = "...",
+                Audience = GetVariable("BRUCKE_MANAGEMENT_API_AUDIENCE"),
+                Scope = "openid email",
+                PhoneNumber = phone
+
+            });
+
+            // Assert
+            token.Should().NotBeNull();
+        }
+
+        [Fact(Skip = "Run manually")]
+        public async Task Can_exchange_email_code_for_access_token()
+        {
+            var authenticationApiClient = new AuthenticationApiClient(GetVariable("AUTH0_AUTHENTICATION_API_URL"));
+
+            // Exchange the code
+            var token = await authenticationApiClient.GetTokenAsync(new PasswordlessEmailTokenRequest
+            {
+                ClientId = GetVariable("AUTH0_PASSWORDLESSDEMO_CLIENT_ID"),
+                ClientSecret = GetVariable("AUTH0_PASSWORDLESSDEMO_CLIENT_SECRET"),
+                Code = "...",
+                Audience = GetVariable("BRUCKE_MANAGEMENT_API_AUDIENCE"),
+                Scope = "openid email",
+                Email = email
+
+            });
+
+            // Assert
+            token.Should().NotBeNull();
+        }
```

Comment:
The above two integration tests can be executed manually to test exchanging tokens.

Reply:
I saw that pattern being used elsewhere (https://github.com/auth0/auth0.net/blob/cbb6847159ccee4139ff12da5c3aa8d6d4021f81/tests/Auth0.AuthenticationApi.IntegrationTests/AccessTokenTests.cs#L108)

The problem here is, to which email or phone number do we send the code when running the integration test? How can we validate it worked if we cant open the email or phone in an automated way?

Reply:
I'm not sure you can - if there is precedence for this and this is the best way to conduct these tests, all good.